packages/noya-renderer/src/components/SketchFileRenderer.tsx

```diff
@@ -62,13 +63,32 @@ const BoundingRect = memo(function BoundingRect({
   rect: Rect;
 }) {
   const CanvasKit = useCanvasKit();
+  const [state] = useApplicationState();
 
   const alignedRect = useMemo(
     () => Primitives.rect(CanvasKit, insetRect(rect, 0.5, 0.5)),
     [CanvasKit, rect],
   );
 
-  return <RCKRect rect={alignedRect} paint={selectionPaint} />;
+  let isShapeALine = false;
+  if (state.selectedObjects.length === 1) {
```

Comment:
I'd recommend moving this stuff out of `BoundingRect`. If we don't need a bounding rect, we can short-circuit before we render `<BoundingRect />`.

We can use the `getSelectedLineLayer` selector I suggest in my other comment

Reply:
created `omitBoundingRect` function